ios/Mappers.swift

```diff
@@ -1075,6 +1075,42 @@ class Mappers {
         default: return .unknown
         }
     }
+  
+
+    class func mapToSurchargeConfiguration(from dict: [String: Any]?) throws -> SurchargeConfiguration? {
+        guard let dict = dict else { return nil }
+        guard let amount = dict["amount"] as? Int else {
```

Comment:
```suggestion
        guard let amount = dict["amount"] as? UInt else {
```

then you shouldn't need the UInt() cast on line 1087